### PR TITLE
Add ext documentation for 3.2 updates

### PR DESCRIPTION
**Pull request by VSEphpbb**
This documentation is for extension developers. It explains what's changed for them with phpBB 3.2 and is based on the sticky topic from here: https://www.phpbb.com/community/viewtopic.php?f=461&t=2349131

### CODE REVIEW

File: development/extensions/new_in_rhea.rst [hunk 1]
```diff
@@ -0,0 +1,489 @@
+========================
+What's New for phpBB 3.2
+========================
+
+Introduction
+============
+
+phpBB 3.2 (rhea) introduces many new and updated components that extensions can take advantage of. Some of these changes may require extensions to make updates to maintain compatibility with phpBB 3.2 (some changes provide a layer of backwards compatibility with 3.1). Extension authors should review the changes documented below to see how their extensions may be affected.
```
Comment (marc1706) [inline development/extensions/new_in_rhea.rst]: Rhea*

File: development/extensions/new_in_rhea.rst [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+ * `Updated INCLUDECSS`_
+ * `Additional Helpers`_
+
+New Language Object
+===================
+
+.. warning::
+    The following applies to phpBB 3.2 and later versions. If you must maintain backwards compatibility with phpBB 3.1, continue using the deprecated language methods from the User object.
+
+A new Language object has been introduced that decouples language functions from the User object.  That is to say, the following language functions now belong to the ``\phpbb\language\language`` class:
```
Comment (marc1706) [inline development/extensions/new_in_rhea.rst]: two spaces: ` object.  That is to`

**Comment by VSEphpbb** [comment]
Updated

File: development/extensions/new_in_rhea.rst [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+.. code-block:: php
+
+    public function create_insert_array($data, $pre_create_data = array())
+    {
+        // prepare some data...
+
+        return parent::create_insert_array($data, $pre_create_data);
+    } 
+
+In phpBB 3.2, the data is now added the the class data property, so it is no longer necessary to use a ``return``, just call the method from the parent class at the end:
```
Comment (Derky) [inline development/extensions/new_in_rhea.rst]: *to the class data property

**Pull request merged**